### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,19 +14,19 @@
...
 - Enables you to better understand the performance of your Support team. It calculates metrics focused on response times, resolution times, and work times for you to analyze. It performs the following actions:
-  - Creates an enriched ticket model with relevant resolution, response time, and other metrics
-  - Produces a historical ticket field history model to see velocity of your tickets over time
-  - Converts metrics to business hours for Zendesk Support Professional or Enterprise users
-  - Calculates SLA policy breaches for Zendesk Support Professional or Enterprise users
+- Creates an enriched ticket model with relevant resolution, response time, and other metrics
+- Produces a historical ticket field history model to see velocity of your tickets over time
+- Converts metrics to business hours for Zendesk Support Professional or Enterprise users
+- Calculates SLA policy breaches for Zendesk Support Professional or Enterprise users
```
Comment (fivetran-joemarkiewicz) [inline README.md:23]:
```suggestion
  - Creates an enriched ticket model with relevant resolution, response time, and other metrics
  - Produces a historical ticket field history model to see velocity of your tickets over time
  - Converts metrics to business hours for Zendesk Support Professional or Enterprise users
  - Calculates SLA policy breaches for Zendesk Support Professional or Enterprise users
```

File: README.md [hunk 2]
```diff
@@ -14,19 +14,19 @@
...
 
-> Note: Tickets from the Zendesk Support Chat channel will not populate in this package as the Fivetran connector does not currently support Chat based tickets. This is a feature request that has been flagged. 
+> Note: Tickets from the Zendesk Support Chat channel will not populate in this package as the Fivetran connector does not currently support Chat based tickets. This is a feature request that has been flagged.
 
 <!--section="zendesk_transformation_model"-->
-The following table provides a detailed list of final models materialized within this package by default. 
+The following table provides a detailed list of final models materialized within this package by default.
 > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_zendesk/#!/overview?g_v=1).
 
 | **model**                    | **description**                                                                                                                                                 |
```
Comment (fivetran-joemarkiewicz) [inline README.md:32]:
```suggestion
The following table provides a detailed list of final tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_zendesk/#!/overview?g_v=1).

| **Table**                    | **Description**                                                                                                                                                 |
```

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-catfritz when you have a chance would you be able to review this PR? This way we can batch it along with the other changes planned for the `v0.17.0` release.

File: README.md [hunk 3]
```diff
@@ -23,42 +23,42 @@
...
 | [zendesk__ticket_field_history](https://fivetran.github.io/dbt_zendesk/#!/model/model.zendesk.zendesk__ticket_field_history) | A daily historical view of the ticket field values defined in the `ticket_field_history_columns` variable and the corresponding updater fields defined in the `ticket_field_history_updater_columns` variable.                                                        |
-| [zendesk__sla_policies](https://fivetran.github.io/dbt_zendesk/#!/model/model.zendesk.zendesk__sla_policies)           | Each record represents an SLA policy event and additional sla breach and achievement metrics. Calendar and business hour SLA breaches are supported.    
+| [zendesk__sla_policies](https://fivetran.github.io/dbt_zendesk/#!/model/model.zendesk.zendesk__sla_policies)           | Each record represents an SLA policy event and additional sla breach and achievement metrics. Calendar and business hour SLA breaches are supported.
 
 Many of the above reports are now configurable for [visualization via Streamlit](https://github.com/fivetran/streamlit_zendesk)! Check out some [sample reports here](https://fivetran-zendesk.streamlit.app/).
```
Comment (fivetran-catfritz) [inline README.md:41]:
```suggestion
Many of the above reports are now configurable for [visualization via Streamlit](https://github.com/fivetran/streamlit_zendesk). Check out some [sample reports here](https://fivetran-zendesk.streamlit.app/).
```

**Review by fivetran-catfritz — CHANGES_REQUESTED** [review]
One of the exclamation marks made it through, but otherwise is in order! @fivetran-joemarkiewicz

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-catfritz thanks for catching that. Let me know if this is good for approval following that change.

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Pull request merged**